tests/unit/llms/providers/test_providers.py

```diff
@@ -88,3 +89,25 @@ def load(self):
     )
 
     assert my_provider.params["provider-param"] == 3
+
+
+def test_llm_provider_params_order_no_provider_type():
+    """Test how missing provider type is handled."""
+    config.init_empty_config()
+    config.dev_config.llm_params = {"provider-param": 3}
+
+    class MyProvider(LLMProvider):
+        @property
+        def default_params(self):
+            return {"provider-param": 1}
```

Comment:
can you also add a default parameter that is not being overwritten and ensure that the default parameter appears in the provider params?  or do you already have that test elsewhere?

Reply:
forget to push? i don't see it

Reply:
added+pushed @bparees